examples/release_draft.yml

```diff
@@ -5,6 +5,9 @@ on:
       tag-name:
         description: 'Name of git tag to be created, and then draft release created. Syntax: "v[0-9]+.[0-9]+.[0-9]+".'
         required: true
+      from-tag-name:
+        description: 'Name of git tag from which to detect changes. Syntax: "v[0-9]+.[0-9]+.[0-9]+".'
```

Comment:
```suggestion
        description: 'Name of the git tag to detect changes from. Default value: latest tag. Syntax: "v[0-9]+.[0-9]+.[0-9]+".'
```

dtto as above

Reply:
Used in commit 669302ec5f2d0d8dab1bca9dc7a44a3967de22af.